packages/design-system-react-native/src/components/Box/README.md

```diff
@@ -0,0 +1,273 @@
+# Box
+
+`Box` is a low level flexbox layout component built on top of the React Native `View` element. It provides a focused set of props that map to Tailwind flexbox utility classes for easy flex-based layout control.
+
+---
+
+## Props
+
+### `flexDirection`
+
+The flexDirection style of the component for controlling the main axis direction.
+
+| TYPE                                     | REQUIRED |
+| :--------------------------------------- | :------- |
+| [BoxFlexDirection](../../types/index.ts) | No       |
+
+### `flexWrap`
+
+The flexWrap style of the component for controlling item wrapping behavior.
+
+| TYPE                                | REQUIRED |
+| :---------------------------------- | :------- |
+| [BoxFlexWrap](../../types/index.ts) | No       |
+
+### `gap`
+
+The gap between the component's children. Use values from the BoxSpacing scale (0-12).
+
+| TYPE   | REQUIRED |
+| :----- | :------- |
+| number | No       |
+
+### `alignItems`
+
+The alignItems style of the component for cross-axis alignment.
+
+| TYPE                                  | REQUIRED |
+| :------------------------------------ | :------- |
+| [BoxAlignItems](../../types/index.ts) | No       |
+
+### `justifyContent`
+
+The justifyContent style of the component for main-axis alignment.
+
+| TYPE                                      | REQUIRED |
+| :---------------------------------------- | :------- |
+| [BoxJustifyContent](../../types/index.ts) | No       |
+
+### `twClassName`
+
+Optional prop to add twrnc overriding classNames for styling beyond flex properties.
+
+| TYPE     | REQUIRED | DEFAULT |
+| :------- | :------- | :------ |
+| `string` | No       | `''`    |
+
+### `style`
+
+Optional style prop forwarded to the underlying `View`.
+
+| TYPE                   | REQUIRED | DEFAULT |
+| :--------------------- | :------- | :------ |
+| `StyleProp<ViewStyle>` | No       | `null`  |
+
+### Additional ViewProps
+
+All other `ViewProps` are passed directly to the underlying `View` component.
+
+---
+
+## Usage
+
+### `flexDirection`
+
+```tsx
+// Horizontal layout (default)
+<Box flexDirection={BoxFlexDirection.Row}>
+  <Text>Item 1</Text>
+  <Text>Item 2</Text>
+</Box>
+
+// Vertical layout
+<Box flexDirection={BoxFlexDirection.Column}>
+  <Text>Item 1</Text>
+  <Text>Item 2</Text>
+</Box>
+```
```

Comment:
Would be good to use the new README format in `docs/component-readme-examples/react-native-readme-example.md`